maps-block-apple.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name:       Block for Apple Maps
  * Plugin URI:        https://github.com/10up/maps-block-apple
  * Description:       An Apple Maps block for the WordPress block editor (Gutenberg).
- * Version:           1.1.1
+ * Version:           1.1.2
```

Comment:
Version numbers will be bumped when we do a release so no need to update those in this PR